src/main.rs

```diff
@@ -205,6 +214,10 @@ where
                                 paused.fetch_xor(true, Ordering::SeqCst);
                                 display_handler.unpark();
                             }
+                            Event::Key(Key::Char('\t')) => {
+                                ui_offset.fetch_add(1, Ordering::SeqCst);
```

Comment:
I'm not thrilled about this... The issue here is that this number will grow infinitely. It's not terrible (we have much larger numbers in the app), but I would rather not do it this way. What do you think of cloning the UI here and adding a toggle method to it? Or maybe you have another idea?

Reply:
I could try to bound the value of the `ui_offset` by using modulo and the number of windows (3 here, but maybe it could change?).

Reply:
That's also a possibility, definitely. Do you think you can get this number from a clone of the ui? that way we won't have to worry about changing this when that number changes, as you say.